packages/lumx-react/src/components/thumbnail/Thumbnail.stories.tsx

```diff
@@ -16,31 +16,37 @@ const numberKnobOtions = {
  * @return simple Thumbnail.
  */
 export const defaultThumbnail = ({ theme }: { theme: Theme }) => {
+    const aspectRatio = select<AspectRatio>('Aspect ratio', AspectRatio, AspectRatio.square, 'Thumbnail');
+    const focusPoint = {
+        x: number('Focus X', 0, numberKnobOptions, 'Thumbnail'),
+        y: number('Focus Y', 0, numberKnobOptions, 'Thumbnail'),
+    };
+    const imageUrl = text('Url image', 'https://i.picsum.photos/id/1001/2400/1400.jpg', 'Thumbnail');
+    const size = select(
+        'Size',
+        {
+            XXS: Size.xxs,
+            // tslint:disable-next-line: object-literal-sort-keys
+            XS: Size.xs,
+            S: Size.s,
+            M: Size.m,
+            L: Size.l,
+            XL: Size.xl,
+            XXL: Size.xxl,
+        },
```

Comment:
(Not a request) This could probably be simplified using lodash `pick`:
```tsx
pick(Size, ['xxs', 'xs', 's', 'm', 'l', 'xl', 'xxl'])
```

Reply:
Or even simpler, an array of sizes also works with the `select` knob
`[Size.xxs, Size.xs, Size.s, Size.m, Size.l, Size.xl, Sizel.xxl]`

Reply:
👍